modules/apiserversource-yaml.adoc

```diff
@@ -145,15 +145,18 @@ spec:
 
 .Verification
 
-To verify that the Kubernetes events were sent to Knative, you can look at the message dumper function logs.
+To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.
 
-. Get the pods by entering the following command:
+. To view the events in a web browser, open the link returned by the following command:
```

Comment:
Please change the numbered list to unordered, as the two methods are not performed sequentially -- I should think most users would do one of the other and not both.

Reply:
How about changing this like here: https://github.com/openshift/openshift-docs/compare/103534fc90ce678a24d7a257f8732e0ffb2c2da1..37cfb46f19e87d2659e953dc1bc3e1781ffc3e4b - to have two verification sections?

Reply:
I should think something this would be fine:

```
.Verification

To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.

. To view the events in a web browser, open the link returned by the following command:
+
[source,terminal]
----
$ oc get ksvc event-display -o jsonpath='{.status.url}'
----
+
. Example browser page
+
image::serverless-apiserversource-eventdisplay-gui.png[Example visualization of ApiServerSource event]

. To see the logs in the terminal, view the event-display logs for the pods by entering the following command:
+
[source,terminal]
----
$ oc logs $(oc get pod -o name | grep event-display) -c user-container
----
+
.Example output
. Example output
[source,terminal]
```

Reply:
That would end up with a numbered list, doesn't it?

> 1. To view..
> 2. Example browser page
> 3. To see the logs..
> 4. Example output

I thought you were saying that it's confusing, and I agree.

Reply:
@cardil Yore'right. I'm sorry.  Should be `*` not `.`.

.Verification

To verify that the Kubernetes events were sent to Knative, you can look at the event-display logs or use web browser to see the events.

* To view the events in a web browser, open the link returned by the following command:
+
[source,terminal]
----
$ oc get ksvc event-display -o jsonpath='{.status.url}'
----
+
.Example browser page
+
image::serverless-apiserversource-eventdisplay-gui.png[Example visualization of ApiServerSource event]

* To see the logs in the terminal, view the event-display logs for the pods by entering the following command:
+
[source,terminal]
----
$ oc logs $(oc get pod -o name | grep event-display) -c user-container
----
+
.Example output
[source,terminal]

Reply:
Done it. There was the same issue in the other file. Fixed that in https://github.com/openshift/openshift-docs/compare/bd304c54ba47f08ec34a1919abe3b78a0c333189..8f4a5f2d3455549ca9909fdbbd2c985f55bc9612